crates/sncast/src/starknet_commands/account/add.rs

```diff
@@ -22,7 +22,7 @@ pub struct Add {
     pub name: String,
 
     /// Address of the account
-    #[clap(short, long, requires = "private_key_input")]
```

Comment:
Command add: Short option names must be unique for each argument, but '-a' is in use by both 'address' and 'account'

Reply:
Reverted bcs of global config future rework